lib/lita/handlers/reviewme.rb

```diff
@@ -118,22 +112,16 @@ def mention_reviewer(response, room: get_room(response))
 
       private
 
-      def next_reviewer(room)
-        ns_redis(room.id).rpoplpush(REDIS_LIST, REDIS_LIST)
-      end
+      def next_reviewer(room, owner = nil)
+        return unless (reviewer = ns_redis(room.id).rpoplpush(REDIS_LIST, REDIS_LIST))
```

Comment:
I think I'm not a huge fan of combining assignment in this guard. I'd probably make that two lines.